pom.xml

```diff
@@ -95,30 +95,27 @@
 		<maven.javadoc.version>3.6.3</maven.javadoc.version>
 		<maven-shade-plugin.version>4.4.3</maven-shade-plugin.version>
 
-		<spring-cloud.version>Hoxton.SR8</spring-cloud.version>
 		<spring.boot.version>3.2.3</spring.boot.version>
 
-		<sonar.exclusions>**/dto/**,**/entity/**,**/exception/**,**/spi/**,**/advice/**,**/config/**,**/constants/**</sonar.exclusions>
-		<sonar.cpd.exclusions>**/dto/**,**/entity/**,**/config/**</sonar.cpd.exclusions>
-
 		<java.version>21</java.version>
-		<validation.api.version>3.0.2</validation.api.version>
 		<jose4j.version>0.6.5</jose4j.version>
-		<kernel.core.version>1.2.1-java21-SNAPSHOT</kernel.core.version>
+		<kernel.core.version>1.2.0.1-B1</kernel.core.version>
 	</properties>
 
 	<modules>
 		<module>certify-service</module>
 		<module>certify-core</module>
-		<module>vci-service-impl</module>
 		<module>certify-integration-api</module>
 	</modules>
 
 	<dependencies>
 		<dependency>
-			<groupId>jakarta.validation</groupId>
-			<artifactId>jakarta.validation-api</artifactId>
-			<version>${validation.api.version}</version>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-validation</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-oauth2-resource-server</artifactId>
```

Comment:
There is no version mentioned, what version will it take ?

Reply:
this will take the version from spring boot -3.2.3